src/pages/Admin/Charity/index.tsx

```diff
@@ -57,7 +57,16 @@ const LINK_GROUPS: { [key in EndowmentType]: LinkGroup[] } = {
     },
   ],
   ast: [
-    ...COMMON,
+    {
```

Comment:
<img width="622" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/75e9079a-efb8-4d92-abbb-f8a9901d112e">

deposit should be shown in ast admin